src/modules/multiple-authors/multiple-authors.php

```diff
@@ -436,6 +437,20 @@ public function print_configure_view()
             );
         }
 
+        /**
+         * Check if Blocks Free or Pro are active
+         */
+        public function pp_blocks_is_active()
+        {
+            if (
+                !MultipleAuthors\Classes\Utils::isPluginActive('advanced-gutenberg.php')
+                && !MultipleAuthors\Classes\Utils::isPluginActive('advanced-gutenberg-pro.php')) {
+            	define('PP_AUTHORS_BLOCKS_INSTALLED', false);
```

Comment:
@htmgarcia I think we should only define the constant here if is not defined. If anyone runs this plugin twice (by mistake or for any reason...) we will break the admin site with a PHP warning about the constant is already defined.

I suggest adding the conditional if constant is defined, at the beginning of the function. And define it once, based on the bool result of the expression you have on this IF right now.

Reply:
This issue is fixed now

Reply:
Thanks @ojopaul I will merge this one and then will merge your changes as well.